src/config.js

```diff
@@ -133,29 +149,20 @@ export default class Config {
         );
     }
     switch (true) {
-      case !!this.inputs.token && !!this.inputs.webhook:
-        core.debug(
-          "Setting the provided token and webhook as secret variables.",
-        );
-        core.setSecret(this.inputs.token);
-        core.setSecret(this.inputs.webhook);
+      case !!core.getInput("token") && !!core.getInput("webhook"):
```

Comment:
This prevents adding both `token` and `webhook` as inputs using `with` while still allowing both environment variables!